### PR TITLE
Update Utilities.groovy with Ubuntu 16.10 and openSUSE 42.1 image names

### PR DESCRIPTION
**Pull request by mellinoe**
@mmitche

### CODE REVIEW

File: jobs/generation/Utilities.groovy [hunk 1]
```diff
@@ -160,6 +160,17 @@ class Utilities {
                                 // For outerloop runs, using Linux kernel version 4.6.2
                                 'outer-linux462': 'auto-auto-ubuntu1604-20160510-20160715outer'
                                 ],
+                            'Ubuntu16.10' :
+                                [
+                                // Generic version label
+                                '' : 'auto-ubuntu1610-20160825',
```
Comment (mmitche) [inline jobs/generation/Utilities.groovy]: I'd just remove this line.  My opinion is now that having empty version names isn't a great idea.

Comment (mellinoe) [inline jobs/generation/Utilities.groovy]: Cool, same for the openSUSE 42.1 section below?

**Comment by mmitche** [comment]
Yeah

**Comment by mellinoe** [comment]
@mmitche Should be good now.

**Comment by mmitche** [comment]
LGTM, did you add the images into the jenkins configuration pages?

**Comment by mellinoe** [comment]
Yep, these should match up with what's on there now.

**Pull request merged**